src/main/python/main/ayab/knitprogress.py

```diff
@@ -44,8 +46,10 @@ class KnitProgress(QTableWidget):
     @date   June 2020
     """
 
-    green = 0xBBCCBB
-    orange = 0xEECC99
+    green = 0x99DD99
+    orange = 0xEECC77
```

Comment:
It doesn't look like this change in colors is related to the topic of this PR, is it?
In any case, changing the color breaks the selection display for right-side stitches, it now displays `stitch Left-x`:
<img width="303" alt="image" src="https://github.com/user-attachments/assets/a1d7fd1a-5e11-4d83-8bcf-af4028bf340b" />

Reply:
Does changing the colour back fix that?

Reply:
there is some fragile code on line 293 of knitprogress.py

Reply:
fixed in https://github.com/AllYarnsAreBeautiful/ayab-desktop/pull/779/commits/274b3529bd2defd340039544e37eb5c77a1deb08